packages/design-system/src/Dialog/styles.module.css

```diff
@@ -0,0 +1,128 @@
+@value variables: "../variables.module.css";
+@value breakpoints_lg,space_3,space_4,fs_3,fs_5,fs_6 from variables;
+
+/* Disable background scroll */
+.noScroll {
+	overflow: hidden;
+}
```

Comment:
This class will be scope by css module (e.g. `noScroll-px6qa1`), and therefore not global.

Make it a global class by placing it into global.css